bundle/Dockerfile

```
@@ -0,0 +1,18 @@
+# Configure the connector runtime version
+ARG RUNTIME_VERSION=0.4.1
```

I believe, the `0.4.2` is the latest one https://hub.docker.com/r/camunda/connectors/tags

```suggestion
ARG RUNTIME_VERSION=0.4.2
```